recipes/Fd/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "fd" %}
+{% set version = "8.38.0" %}
+
+package:
+   name: "{{ name }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: dd255edc891b9addc2e90e2bb536d02ecd716a9236891a674162e15c47c8e088
+
+build:
+  error_overdepending: true
+  error_overlinking: true
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson
+    - ninja
+    - make  # [unix]
+    - pkg-config  # [unix]
+  host:
+    - cfg >=9.0.0
+    - cm >=9.0.1
+    - frv >=4.35.4
+    - libframel >=8.41.1
+  run:
+    - cfg >=9.0.0
+    - cm >=9.0.1
+    - frv >=4.35.4
+    - libframel >=8.41.1
```

Comment:
Do these packages have`run_exports` / should they have one? Otherwise, I expect that we could in future have ABI issues here.

Reply:
Thanks, @xhochy for your review.  No, not all of them yet, but I am in the process of adding `run_exports` to all the dependencies.